specs/www.rst

```diff
@@ -149,6 +149,27 @@ Keys:
   will be ignored. Optional, defaults to ``False``.
 
 
+Server Push
+'''''''''''
+
+Indicates that the server should send a server push (or push promise)
+to the client. This message can be sent at any time after the
+*Response Start* message but before the final *Response Body*
+message. The server should create a new connection if the client
+accepts the push and treat it as if the client had made a request.
```

Comment:
Can HTTP/2 clients reject pushes back to the server? I thought it was always-send style and the client drops it if it doesn't want to use it.

Reply:
I think the client can send a settings frame indicating that it won't accept server push frames.

Reply:
Agreed, and done.